test/Interop/Cxx/templates/swift-class-instantiation-module-interface.swift

```diff
@@ -0,0 +1,10 @@
+// RUN: %empty-directory(%t)
+// RUN: %target-build-swift %S/Inputs/SwiftClassInstantiationModule.swift -module-name SwiftClassInstantiationModule -emit-module -emit-module-path %t/ -I %S/Inputs -Xfrontend -enable-cxx-interop -parse-as-library
+// RUN: %target-swift-ide-test -print-module -module-to-print=SwiftClassInstantiationModule -I %t -I %S/Inputs -source-filename=x -enable-cxx-interop | %FileCheck %s
+
+// CHECK: import MagicWrapper
+// CHECK: import SwiftOnoneSupport
```

Comment:
Is there a specific reason we are checking for `import SwiftOnoneSupport` here? We generally emit it for (almost?) every module, so I don't think we need to check for it explicitly here.

Reply:
No good reason, I just copy-pasted the entire output of `swift-ide-test`. Cleaned up now.